1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+<img src="https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png" width="400"  alt="Sonar screen">
```

Comment:
Use Markdown based [image tagging](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) and not raw HTML.